### PR TITLE
Docs-4410 Update Nextcloud and Admin User Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit removes the non-media Nextcloud article that essentially duplicates the InstallNextCloudMedia.md article content, adds an alias for the other article to the media article. It updates the ManageLocalUsersSCALE.md article to include information about the admin user requirement to set Permit Sudo, and fixes a few things in the rootless login article content.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md [hunk 1]
```diff
@@ -34,34 +38,37 @@ In this procedure you:
...
 
+To set the ffmp binary, include `sudo` in the command and enter the admin password. 
 Go to **System > Shell** and enter these six commands:
 
 ```
-cd /mnt/tank/apps/nextcloud/opt
+cd /mnt/tank/nextcloud/opt
 
-wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz
+sudo wget https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-amd64-static.tar.xz
```
Comment (anodos325) [inline content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md:71]: I don't have much experience with nextcloud stuff. But this seems like a somewhat inappropriate thing to have in official docs. If this is a feature we need in nextcloud then I feel like it should be a feature request to have backend do this in an automated (and correct) way. Having users type things in like this is just a recipe for fat-fingering input and breaking things.

**Comment by micjohnson777** [comment]
The original instructions came from Kris so I don't know if this is
an example or required. I'll ping Kris to see which is the case.

File: content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md (hunk 1 above)
Comment (sonicaj) [reply content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md:71]: @stavros-k can we manage this in the app itself please with an additional question in nextcloud app as a install/update hook perhaps ( safely checking first if this already is setup or not ) ?

Pointing out this to users to do manually is really a recipe for disaster and bug reports where they might have missed some step or done something differently resulting in different outcome or breaking some other part.

Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md:71]: I've added Kris as a reviewer as this article follows instructions he provided last year. I'll make whatever changes we need made. FYI... this content has been out there for at least 6 months already but that doesn't mean it needs to remain in our docs.

Comment (kmoore134) [reply content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md:71]: Agreed we should probably have this scripted in somehow. Nextcloud for reasons doesn't include ffmpeg, but it really is a almost deal-breaker type functionality to not have it available in the image.

Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md]: Removed the section on the ffmeg binary as directed.

Comment (anodos325) [reply content/SCALE/SCALETutorials/Apps/InstallNextCloudMedia.md]: Maybe while the issue is still in people's minds, @micjohnson777 can you copy the original script into a new NAS ticket in Jira and link it to the docs one?

**Review by Mrt134 — APPROVED** [review]
Confirmed with Kris this is good to merge.

**Pull request merged**